Formula/m/mesa.rb

```diff
@@ -99,6 +106,8 @@ def install
         -Dvalgrind=disabled
         -Dtools=drm-shim,etnaviv,freedreno,glsl,nir,nouveau,lima
       ]
+      # Work around fatal error: vtn_generator_ids.h: No such file or directory
+      ENV.deparallelize
```

Comment:
Not sure if this works on `meson` but will need to bypass cache to test out multiple runs.

Reply:
Confirmed that it still happening - https://github.com/Homebrew/homebrew-core/actions/runs/7155534295/job/19484309828?pr=156864#step:4:722

So may try `-j`/`--jobs` instead.